h2o-core/src/main/java/water/webserver/H2OHttpViewImpl.java

```diff
@@ -94,6 +104,30 @@ public boolean gateHandler(HttpServletRequest request, HttpServletResponse respo
   protected boolean isAcceptingRequests() {
     return _acceptRequests;
   }
+
+  /**
+   * @return False if API is supposed to be disabled on a non-leader node and this node is a non-leader node. Otherwise true.
+   */
+  public static boolean isApiEnabledOnThisNode() {
+    final boolean disabledOnNonLeaderNode;
+    final AbstractEmbeddedH2OConfig embeddedConfig = H2O.getEmbeddedH2OConfig();
+    if (embeddedConfig == null) {
+      // If there is no embedded config, use only the sysvar, if present.
+      disabledOnNonLeaderNode = Boolean.getBoolean(DISABLE_NON_LEADER_API);
```

Comment:
whats the point of this sys prop? easier testing? if there is no point for us I would suggest removing it, its less code, and less code is always better

Reply:
No. The point is to have it exposed in case anyone would need it at any time in future - exact use case not specified. I agree, let's delete it.